index.html

```diff
@@ -0,0 +1,346 @@
+            alt=""
+          />
+          <p>2015</p>
+        </div>
+        <p>
+          A daily selection of privately
+          personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="available-articles">
+          <a href="#">html</a>
+          <a href="#">css</a>
+          <a href="#">javaScript</a>
+        </div>
+        <button type="button" class="action-btn">
+          See Project
+        </button>
+      </section>
+
+      <!-- end of showcase2 -->
+
+      <section class="section2 d-flex2">
+        <img
+          src="./images/Snapshoot-Portfolio3.svg"
+          alt=""
+          class="showcase-img"
+        />
+        <h2>Tonic</h2>
+        <div class="article">
+          <p class="canopy">CANOPY</p>
+          <img
+            src="./images/Counter.svg"
+            alt=""
+          />
+          <p>Back End DEv</p>
+          <img
+            src="./images/Counter.svg"
+            alt=""
+          />
+          <p>2015</p>
+        </div>
+        <p>
+          A daily selection of privately
+          personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="available-articles">
+          <a href="#">html</a>
+          <a href="#">css</a>
+          <a href="#">javaScript</a>
+        </div>
+        <button type="button" class="action-btn">
+          See Project
+        </button>
+      </section>
+
+      <!-- end of showcase3 -->
+
+      <section class="section2 d-flex2">
+        <img
+          src="./images/art-printing.svg"
+          alt=""
+          class="showcase-img"
+        />
+        <h2>Multi-Post Stories</h2>
+        <div class="article">
+          <p class="canopy">CANOPY</p>
+          <img
+            src="./images/Counter.svg"
+            alt=""
+          />
+          <p>Back End DEv</p>
+          <img
+            src="./images/Counter.svg"
+            alt=""
+          />
+          <p>2015</p>
+        </div>
+        <p>
+          A daily selection of privately
+          personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="available-articles">
+          <a href="#">html</a>
+          <a href="#">css</a>
+          <a href="#">javaScript</a>
+        </div>
+        <button type="button" class="action-btn">
+          See Project
+        </button>
+      </section>
+      <!-- end of showcase4 -->
+    </article>
+    <!-- end of section -->
+
+    <!-- beginning of about -->
+    <article>
+      <div class="about-me d-flex2">
+        <h2>About Myself</h2>
+        <p>
+          Hello I’m a software developer! I can
+          help you build a product , feature or
+          website Look through some of my work and
+          experience! If you like what you see and
+          have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+        <a class="connect" href="#"
```

Comment:
- [ ] Let's have this heading inside any heading element for more semantics. Good job having them inside `a` tags, but these will be in the DOM as text nodes, instead, let's make them show up as headings for better accessibility as well

Reply:
done